packages/mask/src/plugins/Savings/SNSAdaptor/SavingsDialog.tsx

```diff
@@ -46,6 +49,15 @@ function splitToPair(a: FungibleTokenDetailed[] | undefined) {
     }, [])
 }
 
+
+
+
+function isValidYearnChain<YearnChains>(
+    chainId: string | number | symbol
+  ): chainId is keyof YearnChains {
+    return chainId in YearnChains;
```

Comment:
https://mdn.io/reflect-has

Reply:
Can you clarify pls,

I didn't understand your comment

Reply:
plz do not use `in` keyword